policy/modules/apps/pulseaudio.te

```diff
@@ -149,6 +149,7 @@ userdom_search_user_home_content(pulseaudio_t)
 userdom_manage_user_tmp_dirs(pulseaudio_t)
 userdom_manage_user_tmp_files(pulseaudio_t)
 userdom_manage_user_tmp_sockets(pulseaudio_t)
+userdom_write_all_user_runtime_named_sockets(pulseaudio_t)
```

Comment:
Wouldn't it be better to limit this to only unpriv users?

Reply:
What would be a good way of doing that?

Reply:
Nevermind, it slipped my mind how users were set up.